Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -65,9 +67,13 @@ async def create_new_orders(self, symbol, final_note, state, **kwargs):
                 created_orders.append(created_order)
             if created_orders:
                 return created_orders
+            if has_enough_funds:
```

Comment:
It means that for octobot we can create the order because we have enough funds but the exchange refused to create an order for a reason octobot did not expect (happening at the moment when creating reduce only orders with no open position for example).

Reply:
The idea is to differentiate errors coming for missing funds (which octobot knows), are errors from exchanges when creating an order (octobot did not expect)

Reply:
OK, my question was about 2 points. To understand what you wanted to do, that's now clear to me. The second was about the naming. Here when we read the code we have "if I had enough funds I raise an error", that was not easy to understand what is behind this condition

Reply:
Oh right ! I'll change this :)